### DISCUSSION TITLE
`x448.utils.randomPrivateKey()` returns 57 bytes instead of 56?

### DISCUSSION THREAD

**Discussion by larabr** [paulmillr/noble-curves#87] [Q&A]
I have seen a commit trying to clarify this (https://github.com/paulmillr/noble-curves/commit/77bee0d54e85a5e7a517d492f1524362c0c881c8), but I still don't get it, how is RFC8032 relevant for x448?
Having 57 bytes generated was really unexpected.

**Comment by paulmillr** [comment]
I think you are correct and this one is invalid.

**Comment by paulmillr** [comment]
fixed
